src/svgdevicecontext.cpp

```diff
@@ -472,19 +475,21 @@ void SvgDeviceContext::StartPage()
     if (this->UseGlobalStyling()) {
         m_currentNode = m_currentNode.append_child("style");
         m_currentNode.append_attribute("type") = "text/css";
-        m_currentNode.text().set("g.page-margin{font-family:Times,serif;} "
-                                 //"g.page-margin{background: pink;} "
-                                 //"g.bounding-box{stroke:red; stroke-width:10} "
-                                 //"g.content-bounding-box{stroke:blue; stroke-width:10} "
-                                 "g.ending, g.fing, g.reh, g.tempo{font-weight:bold;} g.dir, g.dynam, "
-                                 "g.mNum{font-style:italic;} g.label{font-weight:normal;} path{stroke:currentColor}");
+        std::string css = "g.page-margin{font-family:Times,serif;} "
+                          //"g.page-margin{background: pink;} "
```

Comment:
I'm wondering if this should be removed or, perhaps better, hidden behind some sort of debug flag?

Reply:
Yeah, not great. I separated it.